pkg/vendir/config/directory.go

```diff
@@ -293,6 +309,17 @@ func (c *DirectoryContentsImage) Lock(lockConfig *LockDirectoryContentsImage) er
 	return nil
 }
 
+func (c *DirectoryContentsImgpkgBundle) Lock(lockConfig *LockDirectoryContentsImgpkgBundle) error {
+	if lockConfig == nil {
+		return fmt.Errorf("Expected image lock configuration to be non-empty")
+	}
+	if len(lockConfig.Image) == 0 {
+		return fmt.Errorf("Expected image Image to be non-empty")
```

Comment:
Should this be "Expected bundle Image to be non-empty"?

Reply:
Addresed. PTAL. Thanks!